Skptak/Lab-Project-FreeRTOS-FAT
# Test formatting bot

Description
-----------
PR to test the formatting bot

Test Steps
-----------

Make a PR
Make a formatting change
Use the /bot run formatting action
Make sure the bot formatted

Checklist:
----------



Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
